### DISCUSSION TITLE
Is there an image resolution multiple / modulus that is ideal?

### DISCUSSION THREAD

**Discussion by ghost** [openMVG/openMVG#2297] [Q&A] [answered]
Padding is added.

![image](https://github.com/openMVG/openMVG/assets/157263624/28673079-c66b-4a42-9b99-6c14c95c5830)

**Comment by pmoulon** [comment]
Hello, could you details what are you exactly doing?
- Are you using the same dataset and downsampling the image, or are you doing something else?

**Reply by ghost** [reply]
The same dataset is used and padding is added to zero the remainder after modulo.

**Reply by pmoulon** [reply]
I still does not understand what you are doing.
Can you illustrate, or details what you are doing?
- if you resize the images to be smaller, why does padding is required...

**Reply by ghost** [reply]
The base image size is kept the same for feature extraction and the intrinsics are fixed afterwards.
The reason I am doing this is because I remembered someone in a image matching competition changed the image resolution multiple to something the feature extractor worked better with and wondered if it is the same case here.

![Train000](https://github.com/openMVG/openMVG/assets/157263624/72cbb9f8-6249-4695-b384-24b2254856f8)

**Reply by pmoulon** [reply, marked as answer]
Could be the case with ML feature extractor. But SIFT has been designed to be scale invariant and so should not be sensible too much to those. As you see on your last test case, things are pretty consistent across the board and I'm happy it is so it show that the feature matching and SfM pipeline is working as expected and not being disturbed by this.

**Reply by ghost** [reply]
Thank you.

**Reply by pmoulon** [reply]
Thank you sharing your tests with us.

**Comment by ghost** [comment]
Multiple resolutions | Sourdough bread 99 images

![chart](https://github.com/openMVG/openMVG/assets/157263624/ad988b77-35c4-4c72-a884-d9372a023639)
